pyproject.toml

```diff
@@ -351,6 +348,17 @@ ignore = [
     "FIX001",
     "FIX002",
     "FA100",
+
+    # Docstrings, maybe to enable later
+    "D100",
+    "D101",
+    "D102",
+    "D103",
+    "D104",
+    "D105",
+    "D106",
+    "D107",
+    "D412",
```

Comment:
For the next person wondering: D100-D107 are for undocumented public symbols and D412 is for "blank-lines-between-header-and-content". So yup, maybe later :)